include/tvm/tir/ir_pass.h

```diff
@@ -385,6 +385,13 @@ Stmt DecorateDeviceScope(Stmt stmt);
  */
 Stmt HoistIfThenElse(Stmt stmt);
 
+/*!
+ * \brief Narrow down PrimExpr datatype in stmt
+ * \param stmt The stmt to do datatype rewrite
+ * \return Transformed stmt.
+ */
+Stmt DataTypeRewrite(Stmt stmt);
```

Comment:
is it better to name it "DataTypeNarrow" ?

Reply:
Renamed.